File: sites-service/src/main/java/com/ironhack/sitesservice/model/Site.java
package com.ironhack.sitesservice.model;

import org.hibernate.annotations.Type;

import javax.persistence.*;
import java.util.List;

@Entity
public class Site {

    /**
     * Properties
     */
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private String name;
    @Type(type = "text")
    private String mapUrl;

    @OneToMany(mappedBy = "site")
    private List<Review> reviewList;

    /**
     * Default constructor
     */
    public Site() {
    }

    /**
     * Constructor: id, name, mapUrl
     */
    public Site(Long id, String name, String mapUrl) {
        this.id = id;
        this.name = name;
        this.mapUrl = mapUrl;
    }

    /**
     * Constructor: name, mapUrl
     */
    public Site(String name, String mapUrl) {
        this.name = name;
        this.mapUrl = mapUrl;
    }

    /**
     * Getters & Setters
     */
    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getMapUrl() {
        return mapUrl;
    }

    public void setMapUrl(String mapUrl) {
        this.mapUrl = mapUrl;
    }

    public List<Review> getReviewList() {
        return reviewList;
    }

    public void setReviewList(List<Review> reviewList) {
        this.reviewList = reviewList;
    }
}

File: grouping-client/src/app/models/site.ts
export class Site {

  constructor(
    private _id: number,
    private _name: string,
    private _mapUrl: string,
    private _tocken: string | null
  ) {}

  public get tocken(): string | null {
    return this._tocken;
  }
  public set tocken(value: string | null) {
    this._tocken = value;
  }

  public get mapUrl(): string {
    return this._mapUrl;
  }
  public set mapUrl(value: string) {
    this._mapUrl = value;
  }
  public get name(): string {
    return this._name;
  }
  public set name(value: string) {
    this._name = value;
  }
  public get id(): number {
    return this._id;
  }
  public set id(value: number) {
    this._id = value;
  }
}